_appmap/web_framework.py

```diff
@@ -109,7 +115,7 @@ def create_appmap_file(
         + " "
         + request_path_info
         + " ("
-        + str(response.status_code)
+        + str(status)
         + ") - "
         + start_time.strftime("%T.%f")[:-3]
```

Comment:
Interesting that pylint doesn't suggest using an f-string here.

Reply:
`consider-using-f-string` is currently disabled. Turning it on doesn't generate that many suggestions, so I'll fix them in a future PR.

Even with it turned on, pylint doesn't see that this could be an f-string. Maybe it's a little too complicated....